docs/user-guide/pods/index.md

```diff
@@ -131,42 +115,25 @@ That approach would provide co-location, but would not provide most of the
 benefits of pods, such as resource sharing, IPC, guaranteed fate sharing, and
 simplified management.
 
-## Durability of pods (or lack thereof)
```

Comment:
There are links to this header in a number of places, so I'd like to leave it here and put a forwarding link below it.